.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,21 @@
+name: Documentation
+description: Report an issue related to the NumPy documentation.
+title: "DOC:"
+labels: [04 - Documentation]
+
+body:
+- type: textarea
+  attributes: 
+    label: "Issue with current documentation:"
+    description: Please make sure to leave a reference to the document/code you're referring to. You can also check the development version of the documentation and see if this issue has already been addressed at https://numpy.org/devdocs.
+  validations:
+    required: false
+
+- type: textarea
+  attributes:
+    label: "Idea or request for content:"
```

Comment:
Hide this heading if there is no response?

Reply:
Is that possible?

Reply:
I had a good look at their documentation and I don't think that's possible yet.